lib/Extension/Completion/CompletionExtension.php

```diff
@@ -99,7 +111,14 @@ private function registerCompletion(ContainerBuilder $container): void
         });
 
         $container->register(LabelFormatter::class, function (Container $container) {
-            return new HelpfulLabelFormatter();
+            switch ($formatter = $container->parameter(self::PARAM_LABEL_FORMATTER)->string()) {
```

Comment:
could use `return match ($formatter) {` here

Reply:
Yeah of course.